### ISSUE TITLE
I'm going for EF Core 3.1 for Cosmos Db mapping. Did I miss something?

### ISSUE THREAD

**Issue by isaacbromberg** [Elfocrash/Cosmonaut#124]
I'm going for EF Core 3.1 for Cosmos Db mapping. Did I miss something?
In a new project, I'm opting for EF Core. The main reason is POCO entities thanks to the good separation of mappings. Or did I miss something in Cosmonaut?

**Comment by Elfocrash** [comment]
I didn't quite get the question. Cosmonaut works with POCO entities as well and you can have mappings using attributes if the property name in the POCO is different from the document.

**Comment by isaacbromberg** [comment]
Yeah, I know that and assumed that differences between classes and document will happen so I will need attributes.

**Comment by Elfocrash** [comment]
Yeah so you can use the [JsonProperty()] attribute to override the name.

**Comment by isaacbromberg** [comment]
After a few months with Cosmos DB and one project with EF core, and one project with Cosmonaut I can summarize that way:

- Ef Core emphasizes mappings
- Cosmonaut emphasizes JSON serialization

Focus on serialization is more straight forward for the document DB. EF mappings feel aligned to relational DB.
